rasp/crawlers/base.py

```diff
@@ -0,0 +1,19 @@
+class BaseBoundedCrawler(object):
+    def run(self):
+        raise NotImplemented("run not implemented for {}"
+                             .format(str(self.__class__.__name__)))
+
+
+class DefaultBoundedCrawler(BaseBoundedCrawler):
```

Comment:
Can you write some quick docs on the crawler?  Also RT what would an UnBounded crawler look like?  Is that just a stateless crawler?